packages/main/src/index.ts

```diff
@@ -97,6 +98,9 @@ if (import.meta.env.PROD) {
 }
 
 let tray: Tray | null = null;
+declare global {
+  let configurationRegistry: ConfigurationRegistry;
+}
```

Comment:
you can declare it like the tray object
```suggestion
let configurationRegistry: ConfigurationRegistry;
```

Reply:
```
let configurationRegistry: ConfigurationRegistry | undefined;
```
It should work as for tray variable
it's also initialized in the same block that were you want to assign configurationRegistry